roombasim/pittras/task/takeoff_task.py

```diff
@@ -1,7 +1,7 @@
 
 import numpy as np
 
-import roombasim.config as cfg
+import roombasim.pittras.config as cfg
```

Comment:
Thanks for the explanation. Can you point me to some reference of how class merging works in Python? I changed this since my linter complained about constants not found. Sorry, I am new to Python :)

Reply:
It is more of a python-hack than a built in feature. Basically, I put a `load` method in the global config class (`roombasim.config`) which can copy in all non-private constants to the global reference: https://github.com/Pitt-RAS/RoombaSIM/blob/master/roombasim/config.py#L12

Then when launching the controller, it will load in the `roombasim.pittras.config` constants via: https://github.com/Pitt-RAS/RoombaSIM/blob/master/roombasim-cli.py#L70

I set it up this way so that our pitt-specific code/constants would be entirely decoupled from the overall simulator

Reply:
The only thing is the python linter doesn't know this happened so it doesn't think those attributes exist.

Reply:
Sounds good. Changing it back.